test/Polly.Specs/Caching/AbsoluteTtlSpecs.cs

```diff
@@ -6,23 +6,32 @@ public class AbsoluteTtlSpecs : IDisposable
     [Fact]
     public void Should_be_able_to_configure_for_near_future_time()
     {
-        Action configure = () => new AbsoluteTtl(DateTimeOffset.UtcNow.Date.AddDays(1));
```

Comment:
An easier way to fix all of these issues without also changing all the formatting is to assign them to the discard variable, i.e. ` _ = new Foo()` etc.

Reply:
I meant more like this, assuming it compiles:

```csharp
Action configure = () => _ = new AbsoluteTtl(DateTimeOffset.UtcNow.Date.AddDays(1));
```

Then the diff is mostly just adding a bunch of `_ = `, than also changing the lines, adding braces etc.

Reply:
Agree. This will be cleaner.

Reply:
Or if you permit using `var` and C#10, we can let the compiler infer the return type to be `Func<AbsoluteTtl>` which also silences CA1806 as I recall.

```cs
var configure = () => new AbsoluteTtl(DateTimeOffset.UtcNow.Date.AddDays(1));
```